server/data/orchestrationApiClient.ts

```diff
@@ -161,13 +161,13 @@ export default class OrchestrationApiClient {
 
   // orchestration-sessions-controller
 
-  async getVisitSessions(offenderNo: string, prisonId: string): Promise<VisitSession[]> {
+  async getVisitSessions(offenderNo: string, prisonId: string, minNumberOfDays: string): Promise<VisitSession[]> {
```

Comment:
Might be preferable to make this an optional parameter so we only override defaults when we have to (and thinking ahead to VB-1179)